src/classes/astandardpaths.cpp

```diff
@@ -1,40 +1,48 @@
 #include "src/classes/astandardpaths.h"
 
-QMap<AStandardPaths::Dirs, QString> AStandardPaths::dirs;
+QMap<AStandardPaths::Directories, QString> AStandardPaths::directories;
 
 void AStandardPaths::setup()
 {
     auto data_location = QStandardPaths::AppDataLocation;
-    dirs = {  // [G]: potential rename to `dirs`
-        {Database, QStandardPaths::writableLocation(data_location)},
-        {Templates, QDir(QStandardPaths::writableLocation(data_location)).filePath("templates")},
-        {DatabaseBackup, QDir(QStandardPaths::writableLocation(data_location)).filePath("backup")}
+    directories = { // [F]: Dir could be ambiguous since it is very similar to QDir
+        {Database, QDir::toNativeSeparators(
+         QStandardPaths::writableLocation(data_location) + '/')},
+        {Templates, QDir::toNativeSeparators(
+         QDir(QStandardPaths::writableLocation(data_location)).filePath(QStringLiteral("templates/")))},
+        {DatabaseBackup, QDir::toNativeSeparators(
+         QDir(QStandardPaths::writableLocation(data_location)).filePath(QStringLiteral("backup/")))}
```

Comment:
That... doesnt look sane. Qt wants `/` for all separators nomatter what system. The toNative just returns the OS dependant separators and it should be used at the edges of the logic that needs paths. AStandardPaths should have the general QDirs that other parts of the program tweak to their specific use. Having already nativeSeparators in the `directories` means new paths need to have the nativeSeparator called as well and we are starting again with clunky code.

Reply:
That's reasonable, preparing a commit!